portal/views/user.py

```diff
@@ -1235,3 +1236,75 @@ def unique_email():
         if user_id != result.id:
             return jsonify(unique=False)
     return jsonify(unique=True)
+
+@user_api.route('/user/<int:user_id>/patient_report', methods=('POST',))
+@oauth.require_oauth()
+def upload_user_document(user_id):
+    """Add a WiserCare Patient Report for the user
+
+    File must be included in the POST call, and must be a valid PDF file.
+    File will be stored on server using uuid as filename; file metadata (including
+    reference uuid) will be stored in the db.
+
+    ---
+    tags:
+      - User
+      - User Document
+      - Patient Report
+    operationId: post_patient_report
+    produces:
+      - application/json
+    parameters:
+      - name: user_id
+        in: path
+        description: TrueNTH user ID
+        required: true
+        type: integer
+        format: int64
+      - in: body
+        name: body
+        schema:
+          id: file
+          required:
```

Comment:
I think you need to add `file` as a property here (see [failing test on TravisCI](https://travis-ci.org/uwcirg/true_nth_usa_portal/jobs/184323681#L378)). 

Something like:
```
properties:
  file:
    type: file
    description: File to upload

```

Also, adding it under a schema may not be strictly necessary. Schemas are mostly used to de-duplicate common elements by referring to existing ones (eg `"$ref": "#/definitions/FHIRPatient"`)